### DISCUSSION TITLE
I need help with downloading the repository folder!

### DISCUSSION THREAD

**Discussion by matt-drummer** [raomin/ESPAltherma#305] [Q&A] [answered]
Hi,

I will apologise in advance for probably being the dumbest person here!

I want to ESPAltherma.

I have platformio but cannot for the life of me find the repository to download, sorry.

I can see all the files but I am not really sure what I am looking for.

I assume they are all in one big folder somewhere that I can download.

I cant seem to see this anywhere.

I would appreciate if somebody could point me to the correct place.

Thank you in advance.

**Comment by raomin** [comment, marked as answer]
Hello @matt-drummer! 
No stupid questions here 🙂
![Screenshot_2023-07-22-09-43-41-070-edit_com android chrome](https://github.com/raomin/ESPAltherma/assets/4460208/15dc28a0-0b3b-40a4-8d03-16fbfbe71a71)

**Comment by matt-drummer** [comment]
Thank you.

I have just arrived at work and I can see it now.

It didn't look like that on my pc at home though, I wouldn't have missed that.

Thank you again.
